<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BattleShips</class>
 <widget class="QMainWindow" name="BattleShips">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1297</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BattleShips</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="table1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>421</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>420</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Play vs Bot</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>421</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="coordinatesInputBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>390</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelInput">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>370</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Input coordinates of your 1x1 ship (example: A1)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="oneCellButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>430</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>input</string>
    </property>
   </widget>
   <widget class="QPushButton" name="severalCellsButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>430</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>input</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shootButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>430</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Shoot!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hostButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>460</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Host game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>500</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1297</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
